pycbc/workflow/coincidence.py

```diff
@@ -66,6 +95,7 @@ def create_node(self, trig_files, bank_file, veto_file, veto_name, template_str,
         node.set_memory(10000)
         node.add_input_opt('--template-bank', bank_file)
         node.add_input_list_opt('--trigger-files', trig_files)
+        node.add_input_list_opt('--statistic-files', stat_files)
```

Comment:
Please add an if statement here to only add the option if the number of stat files is greater than or equal to 1.